src/lib/network/network_autoyast.rb

```diff
@@ -227,6 +227,36 @@ def valid_rename_udev_rule?(rule)
       true
     end
 
+    # Renames a network device represented by given item.
+    #
+    # @param [Integer] item is an item id. See LanItems for detail
+    # @param [String] name_to new device name
+    # @param [String] attr an udev attribute usable in NIC's rule. Currently just
+    #  "KERNELS" or "ATTR{address}" makes sense. This parameter is optional
+    # @param [String] value for the given udev attribute. Optional parameter.
+    def rename_lan_item(item, name_to, attr = nil, key = nil)
+      return if item.nil? || item < 0 || item >= LanItems.Items.size
+      return if name_to.nil? || name_to.empty?
+
+      # selecting according device name is unreliable (selects only in between configured devices)
+      LanItems.current = item
+
+      if !attr.nil? && !key.nil?
+        raise ArgumentError, "Missing udev attr / value" if attr.nil? || key.nil?
```

Comment:
This condition is impossible given the condition one line above it. I demand unit tests.